src/css/App.css

```diff
@@ -0,0 +1,4 @@
+.test {
+    font-family: Cambria, Cochin, Georgia, Times, 'Times New Roman', serif;
+    background: rgba(3, 578, 76, 4);
+  }
```

Comment:
Hello @IssaIan ,
Would you add a new line at the end of file

Reply:
made the changes requested